src/test_utils.jl

```diff
@@ -494,9 +494,42 @@ tangent field.
 
 This function uses [`Mooncake.build_rrule`](@ref) to construct a rule. This will use an
 `rrule!!` if one exists, and derive a rule otherwise.
+
+# Arguments
+- `rng::AbstractRNG`: a random number generator
+- `x...`: the function (first element) and its arguments (the remainder)
+
+# Keyword Arguments
+- `interface_only::Bool=false`: test only that the interface is satisfied, without testing
+    correctness. This should generally be set to `true` (the default value), and only
+    disabled if the testing infrastructure is unable to test correctness for some reason
```

Comment:
Is the default value the wrong way around here?

Reply:
Oooo yes, very good catch.
```suggestion
    correctness. This should generally be set to `false` (the default value), and only
    enabled if the testing infrastructure is unable to test correctness for some reason
```